bld2repo/cli.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import os
+
+from bld2repo import (get_buildrequire_pkgs_from_build, add_rpm_urls, rpm_bulk_download,
+                      create_repo)
+from bld2repo.config import Config
+from bld2repo.utils import get_koji_session
+
+
+def get_arg_parser():
+    description = (
+        "When provided with a build id it will download all buildrequired RPMs"
+        "of a modular koji build into the provided directory and create a repository out of it."
+    )
+    parser = argparse.ArgumentParser("bld2repo", description=description)
+    parser.add_argument("-b", "--build_id", required=True, type=int, help="ID of a koji build.")
+    parser.add_argument("-d", "--result_dir", help="Directory where the rpms are downloaded.")
+    parser.add_argument("-a", "--arch", help=("For which architecture the rpms should be download"
+                                              "ed. The 'noarch' is included automatically."))
+    parser.add_argument("-k", "--koji_host", type=str,
+                        help=("Koji host base url. default: "
+                              "'https://koji.fedoraproject.org/kojihub'"))
+    parser.add_argument("-s", "--koji_storage_host", type=str,
+                        help=("Koji storage storage host base url. default: "
+                              "https://kojipkgs.fedoraproject.org"))
+    return parser
+
+
+def main():
+    parser = get_arg_parser()
+    args = parser.parse_args()
+
+    config = Config()
+    if args.koji_host:
+        config.koji_host = args.koji_host
+
+    if args.koji_storage_host:
+        config.koji_storage_host = args.koji_storage_host
+
+    session = get_koji_session(config)
+
+
+    if args.arch:
+        arch = args.arch
+    else:
+        arch = config.default_arch
```

Comment:
I would rename the `Config.defaulit_arch` variable to just `Config.arch` and set its value in a constructor. If `None` was passed, `"x86_64"` would be used as default. Then we wouldn't have to calculate the architecture when wanting to use it.